execute/table.go

```diff
@@ -22,6 +22,7 @@ const (
 	DefaultStopColLabel  = "_stop"
 	DefaultTimeColLabel  = "_time"
 	DefaultValueColLabel = "_value"
+	DefaultFieldColLabel = "_field"
```

Comment:
This is an influxdb concept so it doesn't end up belonging in the flux repository. There's no such thing as a default field column because fields are exclusive to the influxdb source.

If you want a place for this as a constant, the `stdlib/influxdata/influxdb` package is a great place for it. I think it's ok to add this constant to that package.